isabella-stewart-gardner-museum.csl

```diff
@@ -0,0 +1,332 @@
+          </choose>
+        </group>
+      </if>
+      <else>
+        <text value="Technical report"/>
+      </else>
+    </choose>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true">
+    <layout delimiter="; ">
+      <choose>
+        <if type="review-book personal_communication article" match="any">
+          <group delimiter=" " suffix=".">
+            <text macro="author" suffix=","/>
+            <group delimiter=" ">
+              <text term="in"/>
+              <text variable="container-title" font-style="italic"/>
+              <text variable="issue"/>
+            </group>
+            <group delimiter=" " suffix=".">
+              <label plural="never" variable="page"/>
+              <text variable="page"/>
+            </group>
+            <group delimiter=", ">
+              <group delimiter=": ">
+                <text variable="publisher"/>
+                <text variable="publisher-place"/>
+              </group>
+              <date form="text" date-parts="year-month-day" variable="issued"/>
+            </group>
+          </group>
+        </if>
+        <else>
+          <group delimiter=", ">
+            <group delimiter=" ">
+              <text macro="author-short"/>
+              <text macro="year-date"/>
+            </group>
+            <text macro="citation-locator"/>
+          </group>
+        </else>
+      </choose>
+    </layout>
+  </citation>
+  <bibliography et-al-min="3" et-al-use-first="1">
+    <sort>
+      <key macro="author-or-editor"/>
+      <key macro="year-date"/>
+      <key variable="title"/>
+    </sort>
+    <layout suffix=".">
+      <group display="block" delimiter=" ">
+        <text macro="author-short"/>
+        <text macro="year-date"/>
+      </group>
+      <choose>
+        <if type="book">
+          <text macro="author-or-editor" suffix="."/>
+        </if>
+        <else>
+          <text macro="author" suffix="."/>
+        </else>
+      </choose>
+      <choose>
+        <if type="article-journal">
+          <text variable="title" quotes="true" prefix=" " suffix="."/>
+          <group delimiter=" " prefix=" " suffix=".">
+            <text variable="container-title" font-style="italic"/>
+            <group delimiter=", ">
+              <text variable="volume"/>
+              <group delimiter=" ">
+                <label variable="issue" form="short"/>
+                <text variable="issue"/>
+              </group>
+            </group>
+            <text macro="year-date" prefix="(" suffix="),"/>
+            <text variable="page"/>
+          </group>
+        </if>
+        <else-if type="book">
+          <text variable="title" prefix=" " suffix="." font-style="italic"/>
+          <group delimiter=", " prefix=" " suffix=".">
+            <text macro="volume-and-collection-title"/>
+            <text macro="edition"/>
+          </group>
+          <group delimiter=", " prefix=" (" suffix=")">
```

Comment:
These closing brackets here do not appear for me :(
Looks like this: Walter Isaacson. Steve Jobs. (New York, NY: Simon & Schuster, 2011.
Should be: Walter Isaacson. Steve Jobs. (New York, NY: Simon & Schuster, 2011).